_inc/client/components/dev-card/index.jsx

```diff
@@ -369,6 +376,61 @@ export class DevCard extends React.Component {
 						</label>
 					</li>
 				</ul>
+				<ul>
+					<strong>Scan</strong>
+					<li>
+						<label htmlFor="scanUnavailable">
+							<input
+								type="radio"
+								id="scanUnavailable"
+								value="unavailable"
+								name="unavailable"
+								checked={ 'unavailable' === scanState }
+								onChange={ this.onScanStatusChange }
+							/>
+							Unavailable
+						</label>
+					</li>
+					<li>
+						<label htmlFor="scanProvisioning">
+							<input
+								type="radio"
+								id="scanProvisioning"
+								value="provisioning"
+								name="provisioning"
+								checked={ 'provisioning' === scanState }
+								onChange={ this.onScanStatusChange }
+							/>
+							Provisioning
+						</label>
+					</li>
+					<li>
+						<label htmlFor="scanAwatingCreds">
+							<input
+								type="radio"
+								id="scanAwatingCreds"
+								value="awaiting_credentials"
```

Comment:
I think `awaiting_credentials` is not really a scan stare but a reason for and the scan state is 'unavailable' 
See
<img width="623" alt="Screen Shot 2020-04-27 at 12 50 45 PM" src="https://user-images.githubusercontent.com/115071/80364188-c2e46f80-8885-11ea-8ba0-d6fe6cf80e7a.png">